types/taiko/test/taiko-types-test.ts

```diff
@@ -0,0 +1,4 @@
+import * as taiko from '../../taiko';
+
+// $ExpectType Promise<void>
+taiko.openBrowser();
```

Comment:
Is this file meant to be checked in ? Looks like it is one of the generated files

Reply:
Hi Nivedha,

The files under `types/taiko/test/generated` are generated automatically and they should not be checked-in.

The files under `types/taiko/test/` are written manually. in particular `taiko-types-tests.ts` is a proper source file written by me. This first version is very poor and just meant to get dtslint started, but I am already working on another branch to enrich it with many more tests and it will become something like:

```javascript
import {
  openBrowser,
  closeBrowser,
  switchTo,
  intercept,
  emulateNetwork,
  emulateDevice,
  setViewPort,
} from '../../taiko';

openBrowser(); // $ExpectType Promise<void>
openBrowser({ headless: false }); // $ExpectType Promise<void>
openBrowser({ args: ['--window-size=1440,900'] }); // $ExpectType Promise<void>

// $ExpectType Promise<void>
openBrowser({
  args: [
    '--disable-gpu',
    '--disable-dev-shm-usage',
    '--disable-setuid-sandbox',
    '--no-first-run',
    '--no-sandbox',
    '--no-zygote',
  ],
});

closeBrowser(); // $ExpectType Promise<void>

switchTo(/taiko.dev/); // $ExpectType Promise<void>
switchTo(/Taiko/); // $ExpectType Promise<void>
switchTo(/http(s?):\/\/(www?).google.(com|co.in|co.uk)/); // $ExpectType Promise<void>
switchTo(/Go*gle/); // $ExpectType Promise<void>
switchTo(/google.com/); // $ExpectType Promise<void>
switchTo({ name: 'newyorktimes' }); // $ExpectType Promise<void>
...
```

Reply:
cool, I get that 👍